Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -30,14 +30,13 @@ web page context.
 Content Security Policy declarations can be applied to a TYPO3 website in
 frontend and backend scope with a dedicated API.
 
-To delegate Content Security Policy handling to TYPO3, the scope-specific
-feature flags need to be enabled:
+To delegate Content Security Policy handling to TYPO3 frontend, the feature flag
+:ref:`security.frontend.enforceContentSecurityPolicy <typo3ConfVars_sys_features_security.frontend.enforceContentSecurityPolicy>`
```

Comment:
Why do we write, it needs to be enabled, it is now always enabled?

Reply:
Only backend is enforced now, frontend may be enabled. Look at the prefix ;-)

Reply:
woops